src/libOpenImageIO/compute_test.cpp

```diff
@@ -308,6 +308,14 @@ getargs (int argc, char *argv[])
 
 int main (int argc, char *argv[])
 {
+#if !defined(NDEBUG) || defined(OIIO_TRAVIS) || defined(OIIO_CODECOV)
+    // For the sake of test time, reduce the default iterations for DEBUG,
+    // CI, and code coverage builds. Explicit use of --iters or --trials
+    // will override this, since it comes before the getargs() call.
+    iterations /= 10;
```

Comment:
Many of the tests are multithreaded, so I want the job size big enough for parts to be done in parallel, and for long enough to raise the probability of race conditions changing outcomes. But for debug or CI renders, they sure don't need to be long enough to get accurate timing information. So I figured that speeding them up by 90% was good enough.